infra/src/test/kotlin/fi/fta/geoviite/infra/geometry/TransformationTest.kt

```diff
@@ -1,16 +1,38 @@
 package fi.fta.geoviite.infra.geometry
 
 import fi.fta.geoviite.infra.common.RotationDirection.CCW
+import fi.fta.geoviite.infra.common.Srid
 import fi.fta.geoviite.infra.common.VerticalCoordinateSystem
+import fi.fta.geoviite.infra.dataImport.RATKO_SRID
 import fi.fta.geoviite.infra.geography.HeightTriangle
+import fi.fta.geoviite.infra.geography.KKJtoETRSTriangle
+import fi.fta.geoviite.infra.geography.Transformation
 import fi.fta.geoviite.infra.geography.transformHeightValue
 import fi.fta.geoviite.infra.math.Point
 import fi.fta.geoviite.infra.math.Point3DM
+import fi.fta.geoviite.infra.tracklayout.LAYOUT_SRID
 import fi.fta.geoviite.infra.tracklayout.lengthPoints
 import fi.fta.geoviite.infra.tracklayout.toPointList
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+import java.lang.IllegalArgumentException
 import kotlin.test.assertEquals
 
+val DummyTriangulationNetwork = listOf(
```

Comment:
Tehty yksikkötestejä varten. Not ideal, mutta mikään yksikkötesti ei testaa kolmioverkkoa